src/microbe_stage/CompoundBag.cs

```diff
@@ -39,6 +39,9 @@ public CompoundBag(float capacity)
     /// </summary>
     public float GetCompoundAmount(Compound compound)
     {
+        if (InfCompoundsCheatAndIsUseful(compound))
```

Comment:
This gets called a *ton* by various game systems. I don't particularly like how this now incurs a separate call.
What I'd much, much prefer is to have just a single check in the Microbe run code when it is a player and cheats are on, to add compounds up to the storage limit each frame.

Reply:
Do you mean I should call something like this every frame in microbe:
```
  if Player
    foreach usefulComponents
      AddComponent(9999, usefulComponent)
```
?

Reply:
Almost, it should only add to the compound capacity. Otherwise the player cell would infinitely spew out compounds into the environment.

Reply:
A problem with this aproach is that this won't work if the player has never collected this compound before.

For example, if the player hasn't collected any ammonia before the `CompoundBag.Compounds` Dictionary of the Player does not contain the key `Ammonia`. Therefore, I am unable to know, that the player should have been given ammonia when the cheat is active.
When the player has collected some ammonia the Dictionary will contain the `Ammonia`-Key and I therefore know that I should give the player ammonia.

Reply:
I meant as in add each cloud Compound type from the simulationparameters to always add everything **but** limit the amount so that (new amount + existing amount <= Capacity) in order to prevent the cell from venting the excess. You can skip non-useful compounds for the player. The compound usefulness is set by the process system so no need to collect anything for it to activate.